packages/lasereyes-core/package.json

```diff
@@ -19,7 +19,8 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "prepare": "npm run build"
```

Comment:
This change was required in order for `pnpm` to build the package from our fork when dependency is defined like this:

```
"@omnisat/lasereyes-core": "github:funkybit-labs/lasereyes-mono#rune-transfer-fix&path:/packages/lasereyes-core"
```